src/renderer/vt/tracing.cpp

```diff
@@ -73,7 +73,7 @@ void RenderTracing::TraceString(const std::string_view& instr) const
         const char* const seq = _seq.c_str();
         TraceLoggingWrite(g_hConsoleVtRendererTraceProvider,
                           "VtEngine_TraceString",
-                          TraceLoggingString(seq),
+                          TraceLoggingUtf8String(seq),

```

Comment:
???

Reply:
you didn't actually do any other conpty things in this pull request, so the title confuses me

Reply:
oh it's the trimleft thing isn't it

Reply:
yes it is.

Reply:
also this was attempting to make it show up correct in tvpp. shhh, ignore it.